electron/src/routes.ts

```diff
@@ -458,6 +461,29 @@ module.exports = {
         }).catch(callback);
     },
 
+    checkForPlatformUpdates: (data: {}, callback) => {
+
+        const api = new GitHubClient();
+
+        return api.releases().get().then((result: Array<any>) => {
+
+            let hasUpdate = null;
+
+            if (result && result.length) {
+                result.sort((a, b) => semver.gt(a, b));
+
+                const latestRelease = result[0];
+
+                if (semver.gt(latestRelease.tag_name, app.getVersion())) {
```

Comment:
`app.getVersion()` gives version of the electron runtime, not version of our app.